src/lib/styles/modal.less

```diff
@@ -0,0 +1,37 @@
+/* TODO: create separate modal file */
+/* Seems like we maybe could use this modal stuff for projected content in the modal but we're not right now. Should I keep it? */
```

Comment:
This isn't documented, and I don't think it's used. Seems like it's a remnant from beta.